### PR TITLE
Update Copyright note in README

### PR DESCRIPTION
**Pull request by lluiscampos**
Adding explicit mention to Northern.tech maintenance of the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Please see http://godoc.org/github.com/spacemonkeygo/openssl for more info
 
 ### License
 
-Copyright (C) 2017. See AUTHORS.
+Copyright (C) 2020 Northern.tech AS & others. See AUTHORS.
```
Comment (ghost) [inline README.md:13]: Should we also add ourselves to the AUTHORS file, maybe with contact@northern.tech ?

Comment (lluiscampos) [reply README.md:13]: @kacf I don't really know. Maybe.

But not only AUTHORS, I think we could add a more general note in README explaining that this is a project based on spacemonkeygo and extended by Northern.tech adding functionality for more OpenSSL function calls.

Comment (oleorhagen) [reply README.md:13]: :+1: for the general README note

Comment (ghost) [reply README.md:13]: :+1:

Comment (lluiscampos) [reply README.md:13]: Added. Please review.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,15 @@
 
 Please see http://godoc.org/github.com/spacemonkeygo/openssl for more info
 
+### About
+
+This reposiroty is a fork of [spacemonkeygo OpenSSL](https://github.com/spacemonkeygo/openssl)
```
Comment (merlin-northern) [inline README.md]: they call themselves "[OpenSSL bindings for Go](https://github.com/spacemonkeygo/openssl#openssl-bindings-for-go)", taking into account that it originates from "[Space Monkey Go](https://github.com/spacemonkeygo)", maybe suggested rewording to:
```
This repository is a fork of [Space Monkey Go OpenSSL bindings](https://github.com/spacemonkeygo/openssl)
```
(also small misspell in `reposiroty`)

**Review by merlin-northern — DISMISSED** [review]
besides one comment, lgtm.

File: README.md (hunk 2 above)
Comment (lluiscampos) [reply README.md]: Thanks. Fixed :+1:

**Review by oleorhagen — APPROVED** [review]

**Review by merlin-northern — APPROVED** [review]
lgtm!

**Pull request merged**